scripts/workspace-status.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+echo "STABLE_GIT_COMMIT $(git rev-parse HEAD)"
+echo "STABLE_VERSION $MPDEV_VERSION"
```

Comment:
I still don't understand yet how `MPDEV_VERSION` is specified--is there some definition connecting the github action and its release label?

Reply:
See: https://github.com/GoogleCloudPlatform/marketplace-tools/blob/gibbley/formula-docs/.github/workflows/tag.yaml#L32